tests/ChannelMessagesTest.php

```diff
@@ -269,7 +269,7 @@ public function testPublishConnectionKey() {
         $msg = new Message();
         $msg->name = 'delegatedMsg';
         $msg->data = 'test payload';
-        $msg->connectionKey = 'fake!realtime_key';
+        $msg->connectionKey = 'fake.realtime_key';
```

Comment:
timeout error

you can see the error in the build for the master branch
https://travis-ci.org/ably/ably-php/jobs/218008466#L426